File: src/main/python/common/s3_utils.py
# Databricks notebook source
# mount S3 bucket to cluster
def s3_mount (bucket_name, mount_name):
  try:
    dbutils.fs.mount(f"s3a://{bucket_name}", f"/mnt/{mount_name}")
  except:
    print("WARNING: mount does not exist or is already mounted to cluster")

File: src/main/scala/common/S3Utils.scala
// Databricks notebook source
// mount S3 bucket to cluster
def s3Mount (awsBucketName: String, dbfsMountName: String) {
  var mounted = ""
  try {
    dbutils.fs.mount(s"""s3a://${awsBucketName}""", s"""/mnt/${dbfsMountName}""")
    println(s"""/mnt/${dbfsMountName} successfully mounted""")
  } catch {
    case _: Throwable => println("WARNING: mount does not exist or is already mounted to cluster")
  }
}

// COMMAND ----------


